quantized_mesh_tile/topology.py

```diff
@@ -161,7 +164,7 @@ def _loadGeometry(self, geometrySpec):
         """
         A private method to convert a (E)WKB or (E)WKT to a Shapely geometry.
         """
-        if geometrySpec.startswith('POLYGON Z'):
+        if type(geometrySpec) is str and geometrySpec.startswith('POLYGON Z'):
```

Comment:
then `geometrySpec.startswith('POLYGON Z')`  should be
`geometrySpec.startswith(b'POLYGON Z')`
This should work both on py3 and py2.
Here you're simply ignoring it when in py3.